src/components/SubscriptionList.js

```diff
@@ -1,49 +1,49 @@
-import React, { PropTypes } from 'react';
+import React, { PropTypes } from 'react'
+import Immutable from 'immutable'
+
 import {
   PageHeader,
   Table,
-} from 'react-bootstrap';
-import { SubscriptionItem } from './SubscriptionItem';
+} from 'react-bootstrap'
 
+import { SubscriptionItem } from './SubscriptionItem';
 
-const subscriptionItems = items => (
-  <Table striped bordered condensed hover>
-    <thead>
-      <tr>
-        <th>ID</th>
-        <th>Billing Day of Month</th>
-        <th>Created</th>
-        <th>Price</th>
-        <th>No. of Transactions</th>
-        <th></th>
-      </tr>
-    </thead>
-    <tbody>
-      {items.map(i => <SubscriptionItem key={i.get('id')} subscription={i} />)}
-    </tbody>
-  </Table>
-);
+const NoSubscriptions = props => (
+  <tr><td colSpan='6'><h4>You don't have any recurring donations</h4></td></tr>
+)
 
-const emptyList = () => <div>No subscriptions</div>;
+export const SubscriptionList = props => {
+  let subscriptions
 
-export const SubscriptionList = (props) => {
-  let items;
-  if (!props.subscriptions) {
-    items = emptyList();
+  if(props.subscriptions.size > 0) {
```

Comment:
Looking at the reducer, I noticed we're defaulting to `fromJS({})` instead of `initialState` (which has an empty `Immutable.List` as the default value). Changing the reducer to default to `initialState` will fix this issue.